content/tools/plugins/JenkinsPlugin.md

```diff
@@ -4,70 +4,89 @@ description: "Sample Jenkins plugin using the Contrast Java SDK"
 
-### Global Contrast Vulnerability Security Controls (VSC)
+13. Set the **Number of Allowed Vulnerabilities**. The number is exclusive; if you set a count for "5", it fails on six or more vulnerabilities. This field is **required**.
 
-Once a connection is made, complete the following fields for **Contrast Vulnerability Security Controls**. 
+14. Choose a **Vulnerability Severity** from the options in the dropdown menu (Note, Low, Medium, High or Critical). The plugin sets a filter in the API call for all vulnerabilities greater than or equal to this field. This field is recommended to reduce your results, but not required.
 
-* Select a **Connection** from the dropdown.
-* Set the **Number of Allowed Vulnerabilities**. The number is exclusive; if you set a count for "5", it fails on six or more vulnerabilities. This field is **required**.
-* Choose a **Vulnerability Severity** from the options in the dropdown menu (Note, Low, Medium, High or Critical). The plugin sets a filter in the API call for all vulnerabilities greater than or equal to this field. This field is recommended to reduce your results, but not required. 
-* Choose a **Vulnerability Type** (rule name) from the dropdown menu. If you specify a single rule for which to filter, the plugin checks for the number of vulnerabilities with the rule type and compares it to the count. This field is recommended to reduce your results, but not required. 
-* Choose from the list of **Vulnerability Statuses**. Statuses aren't required, but can be helpful if you want to exclude vulnerabilities with certain statuses - for example, "Not a Problem" - from the results. If you don't select any statuses, the plugin won't filter vulnerabilities by statuses.
+15. Choose a **Vulnerability Type** (rule name) from the dropdown menu. If you specify a single rule for which to filter, the plugin checks for the number of vulnerabilities with the rule type and compares it to the count. This field is recommended to reduce your results, but not required. 
+
+16. Choose from the list of **Vulnerability Statuses**. Statuses aren't required, but can be helpful if you want to include only vulnerabilities with certain statuses - for example, "Suspicious" or "Reported" - from the results. If you don't select any statuses, the plugin won't filter vulnerabilities by statuses.
 
 <a href="assets/images/Jenkins_Global_Vulnerability_Security_Control.png" rel="lightbox" title="VSC configuration"><img class="thumbnail" src="assets/images/Jenkins_Global_Vulnerability_Security_Control.png"/></a>
 
 You can add as many rules as you like. The plugin fails on the **first** bad condition and tells you on which condition it failed.
 
 >**Note**: Even if your build succeeds, the plugin fails the overall build if the test finds a bad condition.
 
-### Contrast Vulnerability Security Controls in a post-build action
 
-Complete the following fields for **Post-Build Actions**.
+### Defining Vulnerability Security Controls (Job Scope)
```

Comment:
@contrast-product I'm going to change Job Scope to Job Level as we reference the scope as "job level" in the plugin.